user-guide/modules/ROOT/pages/common-introduction.adoc

```diff
@@ -27,4 +27,4 @@ Each of these areas is a significant field in its own right, with specific techn
 
 == See Also
 
-* https://pdimov.github.io/boostdep-report/[Boost Dependency Report]
\ No newline at end of file
+* https://pdimov.github.io/boostdep-report/[Boost Dependency Report]
```

Comment:
Sorry. What changed here?

Reply:
Not sure what happened here, a missing newline to end the doc maybe